qlty-cloud/src/token/mod.rs

```diff
@@ -0,0 +1,104 @@
+mod auth_flow;
+
+use crate::Client;
+use anyhow::{Context, Result};
+use keyring::Entry;
+use tracing::warn;
+
+pub const SERVICE: &str = "qlty-cli";
+pub const DEFAULT_USER: &str = "default";
+
+pub struct Token {
+    pub user: String,
+}
+
+impl Default for Token {
+    fn default() -> Self {
+        Self::new(DEFAULT_USER)
```

Comment:
This default user concept was not intuitive to me. Is this trying to represent anonymous usage? If so, can this be represented in the type system as `Option<Token>`?

Reply:
I'm not sure I follow this. A "user" value is required for keyring. It is not optional.

Note that this was present in the original implementation of token.rs, not something I changed-- the only modification was adding a const for the user value so that it can be changed during tests.

Reply:
Ahh thanks for clarifying. Sorry when I read this I was forgot about the key ring user concept and conflated Qlty users with key ring users. 

So in essence I confused myself :) We could optionally just rename this variable to something like DEFAULT_KEYRING_USER